### DISCUSSION TITLE
Bugs on news install ?

### DISCUSSION THREAD

**Discussion by 7zxkv** [OnionUI/Onion#630] [Q&A] [answered]
### Checklist

- [X] I am reporting a bug in the Onion OS (Note that Onion devs can’t fix inherited issues with RA Cores)
- [X] I have reviewed the [Onion Wiki](https://github.com/OnionUI/Onion/wiki) and did not find the answer
- [X] I have searched the existing (open and closed) issues to ensure I do not log a duplicate

### Onions OS Version

4.0.3

### Miyoo Firmware Version

20220419

### Provide a clear and concise description of the issue

Hello, 

1/ I have no games displayed in the official core: supergrafx (the games are in the sgfx directory of the roms directory), any idea?

2/ Otherwise I activated the expert mode, the problem is that I see absolutely nothing once in this mode, black screen and not a single category or image, any idea?

thanks

### Steps to Reproduce

new install...

### Provide any additional context or information that may help us investigate your issue

new install...

**Comment by Aemiii91** [comment, marked as answer]
1) The roms are in `Roms/SGFX` (case-sensitive) and have one of the following extensions: pce|sgx|cue|ccd|chd ?

2) Did you enable any expert systems in Package Manager?

**Comment by 7zxkv** [comment]
OK it's good, thanks you Aemii91. it's hard to find your way on github when you're not used to it ;)

**Reply by Aemiii91** [reply]
Did you find the solution then? You're welcome to post it, to help others :)

**Comment by 7zxkv** [comment]
for 1/ config.json by default is https://pastebin.com/hKY1ajQd, my little modification: https://pastebin.com/miYpiQ23
for 2/ I just didn't see that there were menus when going to the right (and the worst is that I already installed onion before but it didn't jump to my eyes, maybe the menu should be clearer or I need new eyes?^^), so it's me.

**Reply by Aemiii91** [reply]
1) I'm not sure I see the what fixed it? You haven't changed any extensions?
2) True, we need to make this more visible somehow...

**Reply by 7zxkv** [reply]
I took the onion 3.7.1 file I had and put it in the emu/sgfx directory, and the games were visible right away. I put some things from the new into the old (because some things were missing). I agree that the difference is not logical (I don't know if there are invisible characters or not). The only thing I can say is that the installation was new and with the default file my 5 games were not visible, with this file they appeared without doing anything more. So no idea, really.

**Reply by Aemiii91** [reply]
Damn, alright then.. for the next release it is changed a bit, maybe you can test this one?

```json
{
"label":"SuperGfx",
"icon":"../../Icons/sgfx.png",
"iconsel":"../../Icons/sgfx.png",
"launch":"launch.sh",
"rompath":"../../Roms/SGFX",
"imgpath":"../../Roms/SGFX/Imgs",
"gamelist":"../../Roms/SGFX/miyoogamelist.xml",
"useswap":0,
"shortname":0,
"hidebios":1,
"extlist":"pce|sgx|cue|ccd|chd"
}
```

**Reply by 7zxkv** [reply]
I just did it: nothing is visible!
(of course I also delete the cache which is in /roms/sgfx)

**Reply by Aemiii91** [reply]
That's really strange! Which extension does your roms have?

**Reply by 7zxkv** [reply]
zip (.sgx inside)

**Reply by Aemiii91** [reply]
Okay, that doesn't make any sense, since none of the configs have `zip` in the `extlist`. Maybe your old config isn't formatted properly, and it just indexes all files because it doesn't read the `extlist`.

Hmm, adding `zip` to the `extlist` is actually on my to-do, haha..

**Reply by Aemiii91** [reply]
This one should work for you:
```json
{
"label":"SuperGfx",
"icon":"../../Icons/sgfx.png",
"iconsel":"../../Icons/sgfx.png",
"launch":"launch.sh",
"rompath":"../../Roms/SGFX",
"imgpath":"../../Roms/SGFX/Imgs",
"gamelist":"../../Roms/SGFX/miyoogamelist.xml",
"useswap":0,
"shortname":0,
"hidebios":1,
"extlist":"pce|sgx|cue|ccd|chd|zip"
}
```

**Reply by 7zxkv** [reply]
it's good with this file !

**Reply by Aemiii91** [reply]
Great!

**Reply by 7zxkv** [reply]
Hello (and HNY) Aemiii91,

While installing the 4.0.4 update, I noticed that the original file (sgfx) came back and made the roms disappear again. So I had to put back the file you had posted before and it's ok. Maybe it should be officially included?

Another question while I'm at it: The expert mode is only available in "app" now (there is no more a shortcut on the main page?)

In the wiki (https://github.com/OnionUI/Onion/wiki/Installation) it says this:

"Insert your SD Card, boot up your Mini and press SELECT from the Games tab in the main menu to refresh roms."
I confess that I do not see where this is in fact (in short I am not sure to understand the sentence of what we are talking about ^^), but especially is it still used since all the cache files of the roms have been automatically deleted when moving from 4.03 to 4.04 (which is a good thing I think)?

Sincerely.

(sorry for my bad english)

**Reply by Aemiii91** [reply]
1. Yes, 4.0.4 contains all the "bad" things from 4.0.3 - the update was only to support new firmware versions on new devices.

2. You can enable the Expert tab via Tweaks.

3. Just select the Games tab (either hover over it or go into console view) and press SELECT.

**Reply by 7zxkv** [reply]
Thanks Aemiii91 ;)
